release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1438,6 +1467,22 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|Metrics Server
+|Technology Preview
+|General Availability
+|General Availability
+
+|====
+
+[discrete]
+[id="ocp-release-notes-web-console-tech-preview_{context}"]
+=== Web console Technology Preview features
+
+.Web console Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.16 |4.17 |4.18
+
 |{ols-official} in the {product-title} web console
 | Developer Preview
```

Comment:
Should these be developer preview?

Reply:
Good Q. I pinged the OLS writer. Thanks for asking that! I think at some point it was, but now it might be TP I think.